### DISCUSSION TITLE
Bind model with relationship other component with emit

### DISCUSSION THREAD

**Discussion by t-prod** [livewire/livewire#5499] [1. Help]
Hi,

I'm emiting in a modal a product to display more informations. My component is in a modal with this function 👍 

`public function updateModal(Product $product)`

What's the better way to add product relationship like with('stocks) ? How can I do to eager loading.

Thanks

Regards

**Comment by joshhanley** [comment]
@t-prod I would strongly recommend not sending a whole Eloquent Model in an emit, as Livewire needs to serialise the whole model and send it to the front end, which could lead to security issues. I would instead just emit the product ID, then in your method, get the ID, and loads the product and it's relationships from the database then.

But if you still want to do it that way, then you can eager load, after a model has been loaded, using `->load()`, which works similar to `->with()` but it happens after the model is loaded. Like `$product->load('stocks')`.

Hope this helps!
